mod.test.ts

```diff
@@ -736,3 +736,33 @@ Deno.test("test remove", async () => {
   await $`rm -r ${nonEmptyDir}`;
   assertEquals($.fs.existsSync(nonEmptyDir), false);
 });
+
+Deno.test("test mkdir", async () => {
+  const dir = Deno.makeTempDirSync();
```

Comment:
In #40 I just added a withTempDir helper. Let's use that here to ensure that these temporary directories get cleaned up after the test is run.